jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibRunHelper.java

```diff
@@ -55,6 +55,7 @@ static String buildAndRun(
             "-Djib.useOnlyProjectCache=true",
             "-Djib.console=plain",
             "-D_TARGET_IMAGE=" + imageReference,
+            configureInsecureRegistryProperty(imageReference),
```

Comment:
I think this is simple enough to embed
```
"-Djib.allowInsecureRegistries=" + imageReference.startsWith("localhost")
```
, more readable without the method?

Reply:
We should just allow `localhost` and #1551 so that this isn't necessary